bot/src/discord_commands/jibiki_all_search.js

```diff
@@ -0,0 +1,254 @@
+const {throwPublicErrorInfo} = require('./../common/util/errors.js');
+const {Navigation, NavigationChapter} = require('monochrome-bot');
+const axios = require('axios');
```

Comment:
Please specify a timeout for axios, since its default is infinite. Whatever you think is reasonable, I usually do 10000 ms.
```js
const axios = require('axios').create({ timeout: 10000 });
```